electrification.mss

```diff
@@ -209,7 +213,15 @@
        line-color: #97FF2F;
     }
 
-    [frequency=null][frequency!=0][voltage>=15000][voltage<25000] {
+    [frequency=25][voltage=12000] {
+       line-color: @color_12kv_25hz;
+    }
+
+    [frequency=60][voltage=12500] {
+       line-color: @color_12_5kv_60hz;
+    }
+
+    [frequency!=null][frequency!=0][voltage>=15000][voltage<25000] {
```

Comment:
The selector `[frequency!=null][frequency!=0][voltage>=15000][voltage<25000]` and its rule have to be moved upwards (directly before `[frequency=0]`) because the last selector matching for a given feature from the input data decides how the feature is rendered. If selectors appeared in this order, the line would be rendered black because `0 != null` is true:

```css
[frequency = 0] {
    color: red;
}
[frequency =! null] {
    color: black;
}
```

This behaviour makes a railway line with 20 kV 50 Hz to be rendered in yellow (`#F1F100`), not light orange (`#FFCC66`).

Reply:
Done. I just moved the selector for frequency!=null upwards accordingly.